kythe/rust/indexer/src/indexer/analyzers.rs

```diff
@@ -38,6 +38,9 @@ pub struct UnitAnalyzer<'a> {
     file_vnames: HashMap<String, VName>,
     // A map between a file name and its sha256 digest
     file_digests: HashMap<String, String>,
+    // A map between the compiler path of generated files and the path shown in
+    // the save_analysis file
```

Comment:
The compile path is the save_analysis path, no?

Reply:
This refers to the path that the compiler uses to reference files in the save_analysis file